tests/unit/test_data.py

```diff
@@ -56,3 +57,45 @@ def _test_dataloader_drop_last(args, model, hidden_dim):
             model.step()
 
     _test_dataloader_drop_last(args=args, model=model, hidden_dim=hidden_dim)
+
+
+@pytest.mark.parametrize('world_size, total_samples, batch_size, drop_last, result',
+                         [(16,
+                           11788,
+                           3,
+                           True,
+                           245),
```

Comment:
I think it would be better to compute `result` value in the function based on `drop_last` rather than pass as an argument.